src/main/resources/JavadocLexer.g4

```diff
@@ -140,6 +140,7 @@ LINK_LITERAL : 'link'-> pushMode(link);
 LINKPLAIN_LITERAL : 'linkplain' -> pushMode(link);
 VALUE_LITERAL: 'value' -> pushMode(value);
 INHERITDOC_LITERAL : 'inheritDoc' -> pushMode(link);
+SYSTEM_PROPERTY_LITERAL: 'systemProperty' -> pushMode(value);
```

Comment:
https://github.com/checkstyle-GSoC25/javadoc-parser/issues/35
Don't let @nrmancuso see this.
:laughing: